api/parsers/exits.go

```diff
@@ -0,0 +1,103 @@
+package parsers
+
+import (
+	"github.com/gin-gonic/gin"
+	"github.com/hermeznetwork/hermez-node/common"
+	"github.com/hermeznetwork/hermez-node/db/historydb"
+	"github.com/hermeznetwork/tracerr"
+	"gopkg.in/go-playground/validator.v9"
+)
+
+type exitFilter struct {
```

Comment:
Maybe would be a nice idea left those structs within the parsers public to allow a third-party library to reuse them.